network-api/networkapi/wagtailpages/templates/wagtailpages/tags/mini_site_horizontal_nav.html

```diff
@@ -4,7 +4,7 @@
 <div class="mb-4 hidden-sm-down">
   <div class="pb-2 multipage-nav horizontal-nav" id="multipage-nav">
     {% for entry in menu_pages %}
-    <a class="pb-2 multipage-link horizontal-link mr-4 {% if current.title == entry.page.title %}multipage-link-active{% endif %}" href="{% pageurl entry.page %}">{% if entry.depth == 2 %}→ {% endif %}{{ entry.menu_title }}
+    <a class="pb-2 multipage-link horizontal-link mr-4 {% if current.title == entry.page.title %}multipage-link-active{% endif %} {% if entry.restriction != None %}multipage-link-restricted{% endif %}" href="{% pageurl entry.page %}">{% if entry.depth == 2 %}→ {% endif %}{{ entry.menu_title }}
```

Comment:
it feels like we should tap into the string that's in `restriction`, too. rather than setting `multipage-link-restricted`, can we  set something like `multipage-{{restriction}}-restricted` and then define three CSS rules for the three options? (having a 🙈  for private, 🔒 for password locked, and 👥 for groups or something? (aaaand on that note, why does github only offer "family_man_...")